prac_06/guitars.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
```

Comment:
Overall your code is correctly formatted, uses appropriate naming convention, lines are not excessively long and the functions follow SRP (with relevant docstrings).
 
So great job!

The only thing i notice is that the file title docstring is Word Occurrences, which confused me a bit